Comp/r-sas_friedman.qmd

```diff
@@ -0,0 +1,105 @@
+------------------------------------------------------------------------
+
+title: "SAS vs R Non-parametric Analysis - Friedman test"
+
+------------------------------------------------------------------------
```

Comment:
In a quarto file the the page should begin with the following format with just three dashes to denote the beginning and end: